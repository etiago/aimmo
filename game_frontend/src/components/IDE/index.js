import styled from 'styled-components'
import React, { Component, Fragment } from 'react'

export const IDEMenu = styled.nav`
  background-color: pink
  grid-area: ide-menu
`

export const IDEEditor = styled.div`
  background-color: #2F4F4F
  grid-area: ide-editor
`

export const IDEConsole = styled.div`
  background-color: #DDDDDD
  grid-area: ide-console
`

export default class IDE extends Component {
  render () {
    return (
      <Fragment>
        <IDEMenu />
        <IDEEditor />
        <IDEConsole />
      </Fragment>
    )
  }
}
