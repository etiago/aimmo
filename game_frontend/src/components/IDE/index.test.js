/* eslint-env jest */
import React from 'react'
import IDE from 'components/IDE'
import renderer from 'react-test-renderer'

describe('<IDE />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<IDE></IDE>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
