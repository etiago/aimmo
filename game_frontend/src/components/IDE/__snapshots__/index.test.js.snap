// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IDE /> renders correctly 1`] = `
Array [
  <nav
    className="sc-bdVaJa hZrXC"
  />,
  <div
    className="sc-bwzfXH ivIhpv"
  />,
  <div
    className="sc-htpNat crVCkv"
  />,
]
`;
